frontend/layouts/setting.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <v-app class="app-container">
+    <v-main>
+      <v-container>
+        <v-card outlined>
+          <v-toolbar flat>
+            <v-btn　large icon color="primary" :to="backUrl">
```

Comment:
まず、全角空白が混ざってます。
```suggestion
            <v-btn large icon color="primary" :to="backUrl">
```

Vueルーターとの組み合わせにより、to=""に設定したbackURL配下にいるためボタンがアクティブになっています。（つまり、v-btn--activeクラスが付与され、ボタンを常に押しているような見た目になっています）
まだ試していませんが、[こちらの記事](https://teratail.com/questions/205562)が参考になると思います。

Reply:
記事を参考に修正しました。
全角スペースを修正しました。